test/testsymboldatabase.cpp

```diff
@@ -44,13 +44,6 @@
 
 class TestSymbolDatabase;
 
-#define GET_SYMBOL_DB_STD(code) \
```

Comment:
This was unnecessarily added as the settings used in `GET_SYMBOL_DB` already contained `std.cfg`.